src/features/navigator/NarrativeList/NarrativeList.tsx

```diff
@@ -82,14 +76,14 @@ function NarrativeList(props: NarrativeListProps) {
       {props.items.map((item, idx) => {
         return (
           <NarrativeViewItem
-            key={idx}
-            item={item}
             idx={idx}
-            active={idx === selectedIdx}
-            onSelectItem={(idx) => setSelectedIdx(idx)}
-            onUpaChange={(upa) => props.onSelectItem?.(upa)}
+            item={item}
+            key={idx}
+            onUpaChange={
+              (upa) => console.log(upa) /* eslint-disable-line no-console */
+            }
             showVersionDropdown={props.showVersionDropdown}
-          ></NarrativeViewItem>
+          />
         );
       })}
       {hasMoreButton()}
```

Comment:
Agreed, but right now I am just trying to get feature parity. We can refactor things later.

It's also super confusing to see what changes I have needed to make because github (and I suppose git itself) does not understand the file move operation, so the blame is all wrong. Even without that the blame would be wrong since a lot of this code comes directly from another repository.